src/hub.js

```diff
@@ -1115,46 +1113,86 @@ document.addEventListener("DOMContentLoaded", async () => {
     return params;
   };
 
-  const migrateToNewReticulumServer = async deployNotification => {
-    // On Reticulum deploys, reconnect after a random delay until pool + version match deployed version/pool
-    console.log(`Reticulum deploy detected v${deployNotification.ret_version} on ${deployNotification.ret_pool}`);
-    clearInterval(retDeployReconnectInterval);
+  const reconnect = async ({ ret_pool, ret_version }) => {
```

Comment:
Unless I am missing something, this is not doing any sort of reconnecting, so the name is confusing. I think what it's doing is trying to find out if the version we just got a notification for is live, the re-connection happens later.